validator/keymanager/v2/direct/direct.go

```diff
@@ -2,65 +2,162 @@ package direct
 
 import (
 	"context"
-	"errors"
+	"encoding/json"
+	"fmt"
 	"io"
+	"io/ioutil"
 
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/ethereumapis/eth/v1alpha1"
+	"github.com/prysmaticlabs/go-ssz"
+	contract "github.com/prysmaticlabs/prysm/contracts/deposit-contract"
 	"github.com/prysmaticlabs/prysm/shared/bls"
+	"github.com/prysmaticlabs/prysm/shared/depositutil"
+	"github.com/prysmaticlabs/prysm/shared/params"
 	"github.com/sirupsen/logrus"
+	keystorev4 "github.com/wealdtech/go-eth2-wallet-encryptor-keystorev4"
 )
 
 var log = logrus.WithField("prefix", "keymanager-v2")
 
+const (
+	keystoreFileName           = "keystore.json"
+	depositDataFileName        = "deposit_data.ssz"
+	depositTransactionFileName = "deposit_transaction.rlp"
+	eipVersion                 = "EIP-2335"
+)
+
 // Wallet defines a struct which has capabilities and knowledge of how
 // to read and write important accounts-related files to the filesystem.
 // Useful for keymanager to have persistent capabilities for accounts on-disk.
 type Wallet interface {
```

Comment:
Discussed with @farazdagi https://discordapp.com/channels/476244492043812875/483017808658169866/727932592657137704. This is to perform proper dependency injection into keymanager, otherwise you would have an import cycle (keymanager would import accounts/v2 and accounts/v2 would import keymanager)